src/riak_pb_ts_codec.erl

```diff
@@ -36,7 +36,6 @@
          encode_rows/1,
          encode_cells/1,
          decode_rows/1,
-         decode_row/1,
```

Comment:
It is actually used, in riak-erlang-client (which shares riak_pb as a dependency with riak proper).